src/style.css

```
@@ -1 +1,87 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  margin-right: 4px;
+  height: 16px;
+  width: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--0 :nth-child(0) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars--1 :nth-child(1) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars--2 :nth-child(1) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars--2 :nth-child(2) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars--3 :nth-child(1) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars--3 :nth-child(2) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars--3 :nth-child(3) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars--4 :nth-child(1) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars--4 :nth-child(2) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars--4 :nth-child(3) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars--4 :nth-child(4) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars--5 > .stars__star {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}
```

```suggestion
.stars--0 :nth-child(-n),
.stars--1 :nth-child(-n+1),
.stars--2 :nth-child(-n+2),
.stars--3 :nth-child(-n+3),
.stars--4 :nth-child(-n+4),
.stars--5 :nth-child(-n+5) {
    background-image: url('images/star-active.svg');
}
```

Don't duplicate styles and combine selectors with identical styles using comma